src/graphql/schemas/ParseClass.js

```diff
@@ -462,6 +476,56 @@ export class ParseClass {
     };
   }
 
+  graphQLSubscriptionConfig() {
+    const className = this.className;
+    return {
+      name: this.displayName + 'Subscription',
+      description: `Parse Class ${className} Subscription`,
+      fields: () => {
+        const objectFields = this.buildFields(graphQLQueryField);
+        delete objectFields.objectId;
+        delete objectFields.id;
+        return {
+          op : 
+          object : objectFields
+        };
+      },
+      isTypeOf: this.isTypeOf.bind(this),
+    };
+  }
+
+  graphQLEventObjectType(){
+    if(!this.eventObjectType) {
```

Comment:
for all objects the events are shared, this should be a single type exposed , created only once. The graphQL engine will not like creating 1 instance per class of this type